File: src/main/java/testingil/webinar/tddapi/s8/NavigationController.java
package testingil.webinar.tddapi.s8;

import org.springframework.http.MediaType;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/navigator")
public class NavigationController {

	@PostMapping("/destination")
	public void setDestination(@RequestBody Location location) {
	}
	
	@PostMapping("/startpoint")
	public void setStartPoint(@RequestBody Location location) {
	}
	
	@GetMapping(value = "/distance", produces = MediaType.APPLICATION_JSON_VALUE)
	public Distance getDistance() {
		return new Distance();
	}

}

File: Python/s12_re_add_second_test/navigation_controller.py
from flask import Flask

app = Flask(__name__)


@app.route('/nav/destination', methods=['POST'])
def set_destination():
    pass


@app.route('/nav/startpoint', methods=['POST'])
def set_start_point():
    pass


@app.route('/nav/distance', methods=['GET'])
def get_distance():
    pass
